src/outputFunctions.f90

```diff
@@ -356,7 +356,7 @@ subroutine outputSpeciesOfInterest( t, specOutReqNames, allSpeciesConcs )
     end if
 
     if ( first_time .eqv. .true. ) then
-      write (50, '(100A15) ') 't', (trim( specOutReqNames(i) ), i = 1, size( specOutReqNames ))
+      write (50, '(1000A50) ') 't', (trim( specOutReqNames(i) ), i = 1, size( specOutReqNames ))
```

Comment:
Here, and in line 365, could you experiment with removing the repeat number? From memory, and googling, I _think_ this can be just `'(A15) '`, or possibly it needs to be `'((A15)) '`. And the below would be `'(ES15.6E3) '` or `'((ES15.6E3)) '`. If we can remove the repeat number then we of course have can support arbitrary columns.

Apologies that I don't have a working AtChem setup on this laptop to test it quickly, and my usual Fortran reference book is stuck in my office, and I'm not sure how long it will be before I get to lay hands on it again!

Reply:
I've had a bit of a play with this. I think removing the repeat number ends up just adding a newline after each entry. Removing it from line 359 adds newline characters between each species name in the header whereas removing from line 365 adds newlines between each value.